drainer/sync/kafka.go

```diff
@@ -96,6 +96,12 @@ func NewKafka(cfg *DBConfig, tableInfoGetter translator.TableInfoGetter) (*Kafka
 	config.Producer.Retry.Max = 10000
 	config.Producer.Retry.Backoff = 500 * time.Millisecond
 
+	// sasl configuration
+	config.Net.SASL.Enable = cfg.KafkaSASLEnable
+	config.Net.SASL.Handshake = cfg.KafkaSASLHandshake
```

Comment:
According to the documentation:

>You should only set this to false if you're using
>			// a non-Kafka SASL proxy.

I think we'd better make `KafkaSASLHandshake` `true` by default as in `sarama`.